src/mimefactory.rs

```diff
@@ -1381,8 +1392,14 @@ impl MimeFactory {
         } else if msg.viewtype == Viewtype::Webxdc {
             let topic = peer_channels::create_random_topic();
             headers.push(create_iroh_header(context, topic, msg.id).await?);
-            if let Some(json) = context
-                .render_webxdc_status_update_object(msg.id, None)
+            if self.resend {
```

Comment:
Removed `resend` at all. Now updates are sent together with the instance on resending it

Reply:
@link2xt If we want to account the instance size too, then it's a question whether we should do that only for resent instances, or initially too. And if we want to account the instance size only for resent instances, maybe better not to apply the size limit to initially sent instances at all to avoid unwanted message splitting and problems when the second message (containing updates) isn't delivered